ci/code_checks.sh

```diff
@@ -70,11 +70,7 @@ if [[ -z "$CHECK" || "$CHECK" == "docstrings" ]]; then
     RET=$(($RET + $?)) ; echo $MSG "DONE"
 
     MSG='Partially validate docstrings (EX03)' ;  echo $MSG
-    $BASE_DIR/scripts/validate_docstrings.py --format=actions --errors=EX03 --ignore_functions \
-        pandas.Series.plot.line \
-        pandas.Series.to_sql \
-        pandas.read_json \
-        pandas.DataFrame.to_sql # There should be no backslash in the final line, please keep this comment in the last ignored function
+    $BASE_DIR/scripts/validate_docstrings.py --format=actions --errors=EX03
```

Comment:
I think you can remove the whole block, and add `EX03` to the block above, with the errors that are fully validated.

Reply:
Done!

Reply:
Sorry, I don't think I was clear. What I mean is that you delete this line, the one above with the `MSG`, the one below with the `RET`, and in the previous call to `validate_docstrings.py` you add `EX03` to the long list of codes.

Not sure if you understood something else, or maybe the push failed, but in the PR diff I still see this block.